src/subdomains/core/sell-crypto/process/services/buy-fiat.service.ts

```diff
@@ -374,8 +374,9 @@ export class BuyFiatService {
   }
 
   async getSellHistory(userId: number, sellId?: number): Promise<SellHistoryDto[]> {
-    const where = { user: { id: userId }, id: sellId };
-    Util.removeNullFields(where);
+    const where: { user: { id: number }; id?: number } = { user: { id: userId } };
+    if (sellId !== undefined) where.id = sellId;
```

Comment:
Why not just `const where = Util.removeNullFields({ user: { id: userId }, id: sellId });`